src/components/buttons/play-button/play-button.scss

```diff
@@ -0,0 +1,44 @@
+.game-scene {
+  #play-button {
+      position: absolute;
+      top: 23vh;
+      left: 50%;
+      right: 50%;
```

Comment:
Only left: 50% is needed when using transform: translate(-50%, -50%)
right: 50% can be remobed